tests/Spec.hs

```diff
@@ -68,7 +68,7 @@ candidateState pr prBranch prSha prAuthor approvedBy candidateSha =
 -- Types and functions to mock running an action without actually doing anything.
 
 data ActionFlat
-  = ATryIntegrate Text (Branch, Sha)
+  = ATryIntegrate Text (Branch, Sha) Bool
```

Comment:
I would either add a comment to document what the `Bool` stands for, or turn it into a record with a descriptive name.